README.md

```diff
@@ -6,15 +6,18 @@
 
 Talos is a Python variant prioritisation tool which finds variants with diagnostic relevance within large callsets. It incorporates consequence annotation, family structures, participant phenotypes, and existing clinical knowledge to identify variants likely to be relevant to participant phenotypes. It has been configured to do this with high specificity whilst retaining maximal sensitivity to reduce burden on curators.
 
-Analysis consists of two main phases:
+Analysis consists of three main phases:
 
-1. Variant Categorisation
+1. Selection of 'Region of Interest' for the analysis
```

Comment:
Is "region the right word here? Would Loci be a better fit? 
```suggestion
1. Selection of 'Loci of Interest' for the analysis
```

Reply:
Maybe? They're both kinda wrong, as it's really just a gene panel by name based on VEP annotation, I just thought ROI would be the more generic term. I'll swap this out for gene panel to be explicit